tests/model/test_model.py

```diff
@@ -1513,8 +1495,8 @@ def test__handle_reaction_event_remove_reaction(
 
         model._handle_reaction_event(reaction_event)
 
-        # Removed emoji doesn't match, so length remains 1
-        assert len(model.index['messages'][event_message_id]['reactions']) == 1
+        end_reactions = model.index['messages'][event_message_id]['reactions']
+        assert len(end_reactions) == expected_number_after
```

Comment:
"reactions_after_event" might be a better indication than "end_reactions" here?

Reply:
I agree we could use that. I normally do err towards more verbose, particularly in code, but when we're pushed for space in tests and the variable is used on the next line this is not so critical.

There is always the option we could move towards a longer line length like ~88, but we might want to wait for any potential transition to using something like `black`.